src/app/core/services/split-expense.service.spec.ts

```
@@ -755,12 +773,12 @@ describe('SplitExpenseService', () => {
     const expectedPayload = [
       {
         expense_id: 'txn1',
-        comment: splitExpenseService.prependPolicyViolationMessage + 'First reason',
+        comment: prependPolicyViolationMessage + 'First reason',
```

Biome (1.9.4)

[error] Template literals are preferred over string concatenation.

Unsafe fix: Use a template literal.


(lint/style/useTemplate)